src/sentry/tasks/post_process.py

```diff
@@ -1440,6 +1441,7 @@ def detect_new_escalation(job: PostProcessJob):
                 # TODO(snigdha): reuse manage_issue_states when we allow escalating from other statuses
                 add_group_to_inbox(group, GroupInboxReason.ESCALATING)
                 record_group_history(group, GroupHistoryStatus.ESCALATING)
+                auto_update_priority(group, PriorityChangeReason.ESCALATING)
```

Comment:
Since we modified `post_process` should we add/modify a test in `test_post_process`? Just to ensure that it works end-to-end (and I'm scared of messing up post_process 😬)